proto/http.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package edgehog_device_forwarder_proto;
```

Comment:
Package names should based on the path of the file containing the protocol buffer type definitions and should be use dot-separated namespace components.
For example
- path : `edgehog/device/forwarder`, package: `edgehog.device.forwarder`

Reply:
Done